netbsd/Platform.c

```diff
@@ -382,9 +386,27 @@ bool Platform_getDiskIO(DiskIOData* data) {
 }
 
 bool Platform_getNetworkIO(NetworkIOData* data) {
-   // TODO
-   (void)data;
-   return false;
+   struct ifaddrs* ifaddrs = NULL;
+
+   if (getifaddrs(&ifaddrs) != 0)
+      CRT_fatalError("Error getting network interface data");
```

Comment:
Wouldn't a normal fail based on returning `false` suffice? Or is this based on other platform's behavior?